### PR TITLE
KFSPTS-7178 moving utility classes to service architecture

### PR DESCRIPTION
**Pull request by jhulslander**

### CODE REVIEW

**Comment by jhulslander** [comment]
This is ready for review, it just moves a few remaining utility classes into services.  They will be refactored when we turn on the end points they are used in.

File: src/main/java/edu/cornell/kfs/paymentworks/batch/PaymentWorksUploadSuppliersStep.java [hunk 1]
```diff
@@ -33,9 +33,10 @@
 import edu.cornell.kfs.paymentworks.batch.report.SupplierUploadSummaryLine;
 import edu.cornell.kfs.paymentworks.businessobject.PaymentWorksVendor;
 import edu.cornell.kfs.paymentworks.service.PaymentWorksKfsService;
+import edu.cornell.kfs.paymentworks.service.PaymentWorksSupplierConversionService;
 import edu.cornell.kfs.paymentworks.service.PaymentWorksVendorService;
 import edu.cornell.kfs.paymentworks.service.PaymentWorksWebService;
-import edu.cornell.kfs.paymentworks.util.PaymentWorksSupplierConversionUtil;
+import edu.cornell.kfs.paymentworks.service.impl.PaymentWorksSupplierConversionServiceImpl;
```
Comment (nkimble) [inline src/main/java/edu/cornell/kfs/paymentworks/batch/PaymentWorksUploadSuppliersStep.java]: edu.cornell.kfs.paymentworks.service.PaymentWorksSupplierConversionService is imported on line 36. 
Why is the implementation for this service also being imported?

File: src/main/java/edu/cornell/kfs/paymentworks/service/impl/PaymentWorksWebServiceImpl.java [hunk 2]
```diff
@@ -57,9 +57,7 @@
 
 @Transactional
 public class PaymentWorksWebServiceImpl implements PaymentWorksWebService {
-
-	private static final org.apache.log4j.Logger LOG = org.apache.log4j.Logger
-			.getLogger(PaymentWorksWebServiceImpl.class);
+	private static final org.apache.log4j.Logger LOG = org.apache.log4j.Logger.getLogger(PaymentWorksWebServiceImpl.class);
 
 	private static final String NEW_VENDOR_REQUEST_PENDING_VENDORS = "new-vendor-requests/";
 	private static final String NEW_VENDOR_REQUEST_UPDATE_STATUS = "new-vendor-requests/bulk/";
```
Comment (nkimble) [inline src/main/java/edu/cornell/kfs/paymentworks/service/impl/PaymentWorksWebServiceImpl.java]: I realize this code was not modified with these changes but should directory structures be hard coded this way?  Wouldn't it be better to have these two items in a properties file that allows for easier modification later on?

File: src/main/java/edu/cornell/kfs/paymentworks/batch/PaymentWorksUploadSuppliersStep.java (hunk 1 above)
Comment (jhulslander) [inline src/main/java/edu/cornell/kfs/paymentworks/batch/PaymentWorksUploadSuppliersStep.java]: Good catch, I will remove this import

File: src/main/java/edu/cornell/kfs/paymentworks/service/impl/PaymentWorksWebServiceImpl.java (hunk 2 above)
Comment (jhulslander) [inline src/main/java/edu/cornell/kfs/paymentworks/service/impl/PaymentWorksWebServiceImpl.java]: these are URL rest call paths. Perhaps they would be better be handled in another way.  I will put a story in the backlog to review that, and do it consistently with what we are doing amazon and concur.

**Comment by nkimble** [comment]
Overall the changes look good. Once a few questions I had are answered, I will re-review.

**Comment by jhulslander** [comment]
retest this please

**Comment by jhulslander** [comment]
This is ready for review again .... I removed the un-needed import and answered teh other two questions.

**Comment by nkimble** [comment]
Unneeded import was removed. 
Questions were answered as being issues that would be addressed during later refactoring efforts of those services when the developer turns on the end points where/when the service are used. Merging code based on that.

**Pull request merged**